web/client/epics/timeline.js

```diff
@@ -70,7 +70,14 @@ const toISOString = date => isString(date) ? date : date.toISOString();
  * @returns a stream of data with histogram and/or domain values
  */
 const loadRangeData = (id, timeData, getState) => {
-    const {range, resolution} = roundRangeResolution( rangeSelector(getState()), MAX_HISTOGRAM);
+    /**
+     * when there is no timeline state rangeSelector(getState()) returns undefiend, so instead we use the timeData[id] range
+     */
+    const dataRange = timeData.domain.split('--');
```

Comment:
Server may return also a list of values (corner cases, 1 element only or an empty list) you must consider also these 2 cases.
I suggest: 
 - in the case of list to get the first and the last
 - in case of single value range should be something arbitrary like 1 day before - 1 day after the single value
 - No data. 1 day before 1 day after today

Reply:
Good Point, querying the histogram is not needed in that case. So maybe initializing range is not needed. 
We can only check if single values are shown. I'll check if an issue is present for this point.